<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PoldiWindow</class>
 <widget class="QMainWindow" name="poldiWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Poldi UI</string>
  </property>
  <widget class="QWidget" name="widgetMainRow">
   <layout class="QVBoxLayout" name="layoutBase">
   </layout>
   <zorder>splitterList</zorder>
   <zorder></zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
 </widget>
 <tabstops>
 </tabstops>
 <resources/>
 <connections/>
</ui>
